### ISSUE TITLE
[3.0]CONTAINS empty list or empty dict return unexpected result

### ISSUE THREAD

**Issue by lightjiao** [orientechnologies/orientdb#8653]
## Expected behavior  
```
select from v where test contains []
```
return record which `test` value is `[[]]`

## Actual behavior  
looks like return all the records that have `test` field

## Steps to reproduce  
```
create 4 records
create vertex v content {"name": "jack", "age": 22}
create vertex v content {"name": "rose", "age": 22, "test": [[]]}
create vertex v content {"name": "pete", "age": 22, "test": [{}]}
create vertex v content {"name": "david", "age": 22, "test": ["hello"]}
```

select record
```
select from v where test contains []
return three record ( rose, pete and david ), looks like return all the records that have `test` field

select from v where test contains {}
return one record and looks work fine.
```

Thanks reading

**Comment by luigidellaquila** [comment]
Hi @lightjiao 

I'd say it's a bug, I'll check it asap

Thanks

Luigi

**Comment by lightjiao** [comment]
Hi @luigidellaquila 
Is there any fix about this issue ?

Thanks.

**Comment by luigidellaquila** [comment]
Hi @lightjiao 

Sorry, I'm still working on it, I'll keep you updated

Thanks

Luigi

**Comment by luigidellaquila** [comment]
Hi @lightjiao 

I pushed a fix for this problem, it will be released with v 3.0.12

Thanks

Luigi
